controllers/reconcile.go

```diff
@@ -112,6 +112,14 @@ func populateContainerVolumeMounts(frontendEnvironment *crd.FrontendEnvironment)
 		MountPath: "/opt/app-root/src/build/stable/operator-generated",
 	})
 
+	if frontend.Name != "chrome" {
+		volumeMounts = append(volumeMounts, v1.VolumeMount{
+			Name:      "caddy",
+			MountPath: "/opt/app-root/src/Caddyfile",
```

Comment:
Note for self - I probably need to look at this next week - I was testing locally with the chrome deployment, but I'm not sure all the other Caddyfiles are in this dir.

Reply:
@psav @Hyperkid123 lets sync up next week - I was also thinking about if we should allow disabling this behavior via the frontend.yaml of each app (and maybe allowing for customizing the target directory). But I also get if we want to keep it the same for everyone.